docs/src/howto-guides/007_test_connectiom.md

```diff
@@ -0,0 +1,40 @@
+# How to test the connection to cloud?
+
+We provide a way to test the connection from your device to a cloud provider.
+You can call this connection check function by
+
+```shell
+sudo tedge connect <cloud> --test
+```
+
+It returns exit code 0 if the connection check is successful, otherwise, 1.
+
+Also, this test is a part of the `tedge connect <cloud>` process.
+
+## What does the test do?
+
```

Comment:
I feel we can just add a few lines on what the `test` command does in a generic manner: Send a message to the cloud and wait for a response. The subsequent sections can detail out the cloud specific behaviour, if needed.